StudCity.Frontend/src/pages/Profile/EditProfile/ProfileSettings/index.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+import SelectField from "../../../../UI/fields/SelectField";
+import ToggleContainerText from "../../../../UI/ToggleContainerText";
+import Button from "../../../../UI/Button";
+
+export default function ProfileSettings() {
+	return (
+		<div className="w-10/12 min-h-fit max-h-full mx-auto ml-6 flex flex-col gap-5">
+			<Accord isExpanded={true}>
```

Comment:
I think it's a bad idea to do it that way!

Why not make components if they were called Accordion, in which the isExpanded title and children properties would wake up, this component should be more reusable, and we can transfer your toggles to another component, they can be useful